src/utilities/time.ts

```diff
@@ -64,11 +64,11 @@ export function convertDurationStringToUs(durationString: string): number | neve
 }
 
 /**
- * Get a number value in microseconds given a string duration of the format '2y 318d 6h 16m 19s 200ms 0us'.
+ * Get a formatted duration string given a duration in microseconds.
  * @example convertUsToDurationString(90577779200000) -> '2y 318d 6h 16m 19s 200ms 0us'
```

Comment:
Since `includeZeros` is defaulted to `false` should this `@example` exclude `0us`?